### PR TITLE
Ot217 57

### PR DESCRIPTION
**Pull request by filburt88**
Activities-Page created - with pagination and styles.
* Service created to bring data from API/activities and rendering cards
* Activities-card added with Inputs, to add reusability.
* AOS library added just for styling effect on rendering view
* material module added (this must be erased when launching to production)
* Activity Interface created.

### CODE REVIEW

File: src/app/material/material.module.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import { NgModule } from '@angular/core';
```
Comment (anagon88) [inline src/app/material/material.module.ts:1]: Estas seguro que se necesita todos estos modulos? Agregar solo los que se van usando

**Review by anagon88 — CHANGES_REQUESTED** [review]
Mover la historia de usuario a la columna Code Review (tercer columna) y adjuntar a este pull request la evidencia (screenshots la funcionalidad terminada)

File: src/app/material/material.module.ts (hunk 1 above)
Comment (filburt88) [reply src/app/material/material.module.ts:1]: Ya borré los módulos que no usaba.

**Comment by filburt88** [comment]
Componente funcionando:

OT217-57 Mobile View

OT217-57 desktop-view

**Review by anagon88 — APPROVED** [review]

**Pull request merged**